crates/miden-lib/src/account/interface/component.rs

```diff
@@ -31,11 +31,11 @@ pub enum AccountComponentInterface {
     /// slot has a format of `[max_supply, faucet_decimals, token_symbol, 0]`.
     BasicFungibleFaucet(u8),
     /// Exposes procedures from the
-    /// [`RpoFalcon512`][crate::account::auth::RpoFalcon512] module.
+    /// [`AuthRpoFalcon512`][crate::account::auth::AuthRpoFalcon512] module.
     ///
-    /// Internal value holds the storage slot index where the public key for the RpoFalcon512
+    /// Internal value holds the storage slot index where the public key for the AuthRpoFalcon512
```

Comment:
I'm not sure we need to update the name here, since it is the name of the authentication scheme, which remains the same.